lib/active_admin/views/pages/index.rb

```diff
@@ -130,7 +130,7 @@ def render_index
           pagination_total = config[:pagination_total].nil? ? true : config[:pagination_total]
 
           paginated_collection(collection, :entry_name     => active_admin_config.resource_label,
-                                           :entries_name   => active_admin_config.plural_resource_label(:count => collection_size),
+                                           :entries_name   => active_admin_config.plural_resource_label(:count => collection_size(collection)),
```

Comment:
Line is too long. [143/80]<br>Unnecessary spacing detected.<br>Use the new Ruby 1.9 hash syntax.